starters/next/services/nacelleClient.js

```diff
@@ -0,0 +1,10 @@
+import { Storefront } from '@nacelle/storefront-sdk';
```

Comment:
This works, but it might be good to stick to using the default export, just for the sake of staying consistent with [the README for `@nacelle/storefront-sdk`](https://www.npmjs.com/package/@nacelle/storefront-sdk)

```suggestion
import Storefront from '@nacelle/storefront-sdk';
```

<img width="600" alt="storefront-sdk import" src="https://user-images.githubusercontent.com/5732000/151393660-e636cf75-1d0e-4acf-9994-b9049edfb254.png">

Reply:
I don't get an error when using this method, but eslint does flag it as being incorrect. Does appear to work though.

<img width="725" alt="Screen Shot 2022-01-27 at 11 18 12 AM" src="https://user-images.githubusercontent.com/1411409/151428665-d785adc1-6cc4-4108-8aa7-4d89c0bf4729.png">

Reply:
Either way I think we may want to just satisfy ESLint and use the named export

Reply:
That's odd, the default export should be totally valid:

https://github.com/getnacelle/nacelle-storefront-sdk/blob/dc71bb4c3cf1a608475b2372a1b4abbdfd193b72/src/index.ts#L1-L7